yccp/utils.py

```diff
@@ -24,6 +25,9 @@ def get_recursive(dct, path, default=None, sep="/"):
     """
```

Comment:
Actually, we should probably move `default` into `**kwargs` and explicitly check for its presence.

If it was not specified: Raise an exception.
If it was specified, return whatever it is (including `None`).